### PR TITLE
Fully enable bags-list pallet in polkadot runtime

### PR DESCRIPTION
**Pull request by kianenigma**
Some context on my last testing: 

1. I rebagged all of the nodes in Kusama today: https://kusama.subscan.io/extrinsic/10803611-2
2. Also did the same for polkadot: https://polkadot.subscan.io/extrinsic/8419931-4

Also ran all of my scripts and tests for checking bags-list, and it all seems good. Once this is in, and working, we can lift the `22_500` limit on the staking side, namely the `MaxNominators` storage items, which can be done through a governance motion, not a full runtime upgrade.

### CODE REVIEW

**Review by emostov — APPROVED** [review]

**Review by shawntabrizi — APPROVED** [review]

**Comment by shawntabrizi** [comment]
does this need a new benchmark?

**Comment by kianenigma** [comment]
> does this need a new benchmark?

Aye

File: runtime/kusama/src/weights/pallet_staking.rs [hunk 1]
```diff
@@ -343,13 +347,13 @@ impl<T: frame_system::Config> pallet_staking::WeightInfo for WeightInfo<T> {
...
-			// Standard Error: 724_000
-			.saturating_add((300_240_000 as Weight).saturating_mul(v as Weight))
-			// Standard Error: 36_000
-			.saturating_add((49_412_000 as Weight).saturating_mul(n as Weight))
-			.saturating_add(T::DbWeight::get().reads(10 as Weight))
+			// Standard Error: 865_000
+			.saturating_add((235_921_000 as Weight).saturating_mul(v as Weight))
+			// Standard Error: 43_000
+			.saturating_add((37_925_000 as Weight).saturating_mul(n as Weight))
+			.saturating_add(T::DbWeight::get().reads(192 as Weight))
```
Comment (kianenigma) [inline runtime/kusama/src/weights/pallet_staking.rs:354]: hmm have to look into this one.

Comment (kianenigma) [reply runtime/kusama/src/weights/pallet_staking.rs:354]: right, well, we need to lookup all bags upon each new era, and the number of bags is always bounded. Should be fine.

**Comment by kianenigma** [comment]
bot merge

**Pull request merged**